src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,20 @@
+status=error
+name=PropertiesConfig
+
+filters=threshold
+
+filter.threshold.type=ThresholdFilter
+filter.threshold.level=debug
```

Comment:
is there a reason to left "debug" level?

Reply:
Yes, If I delete the line `filter.threshold.level=debug` the server won't start

Reply:
but there are at leats 5 leves,  what is the reason to use "debug" level?

Reply:
Because with this level we can log what is happening in our development time, but I search that for us is better to use the level `info` because this level is used to log a process that we develop.
If we keep the `debug` level we are going to watch every time a log (e.g. like a server status, our logs, etc).
Would you like that do I change the level?

Reply:
I would recommend create a env variable in order to have control on log level and adding `info` level as default.
As you said, for development purposes it's a good idea have `debug` level but for higher environments a huge amount of logs is not recommended because of performance.
Please apply changes.

Reply:
Ok, let me do those changes